engine/browserParser.js

```diff
@@ -1,18 +1,20 @@
 import cache from './cache.js';
 
 export default class BrowserParser {
-  results = {
```

Comment:
stage-0 includes stage-1. seems like the linter doesn't know about stage-1, or specifically the class properties.